src/test/java/org/broadinstitute/hellbender/tools/walkers/fasta/ShiftFastaTest.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.tools.walkers.fasta;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.testutils.BaseTest;
+import org.broadinstitute.hellbender.testutils.FastaTestUtils;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class ShiftFastaTest extends CommandLineProgramTest {
```

Comment:
Rename to "ShiftFastaIntegrationTest" the test suite uses reflection to decide where to run tests so naming them correctly is important.